frontend/plan/src/reducers/schedule.js

```diff
@@ -116,7 +228,15 @@ export const schedule = (state = initialState, action) => {
                 schedules: {
                     ...state.schedules,
                     [nextAvailable(action.scheduleName, state.schedules)]:
-                        state.schedules[action.scheduleName],
+                        {
+                            ...withoutId(state.schedules[action.scheduleName]),
+                            pushedToBackend: false,
+                            updated_at: Date.now(),
+                            isNew: {
```

Comment:
Why is `isNew` an object here instead of a boolean?

Reply:
renamed for clarity